docs/_docs/templates.md

```diff
@@ -6,8 +6,8 @@ permalink: /docs/templates/
 Jekyll uses the [Liquid](https://shopify.github.io/liquid/) templating language to
 process templates. All of the standard Liquid [tags](https://shopify.github.io/liquid/tags/control-flow/) and
 [filters](https://shopify.github.io/liquid/filters/abs/) are
-supported. Jekyll even adds a few handy filters and tags of its own to make
-common tasks easier.
+supported. Jekyll even adds four new tags and even more handy filters of its own to make
```

Comment:
No need to mention they are four tags. This number could change, and we won't remember to update this, we could link to the aforementioned tags if we really wanna be helpful.